libs/nio/src/main/java/org/elasticsearch/nio/EventHandler.java

```diff
@@ -159,12 +159,11 @@ protected void listenerException(Exception exception) {
     }
 
     /**
-     * This method is called after ready events (READ, ACCEPT, WRITE, CONNECT) have been handled for a
-     * channel.
+     * This method is called after events (READ, WRITE, CONNECT) have been handled for a channel.
      *
      * @param context that was handled
      */
-    protected void postHandling(SocketChannelContext context) {
+    protected void postSocketChannelHandling(SocketChannelContext context) {
```

Comment:
I liked postHandling better since `SocketChannel` is kind of part of the signature due to the argument? WDYT?